build.sbt

```diff
@@ -2,17 +2,30 @@ import Settings._
 import Testing._
 
 
+lazy val rawlsUtil = project.in(file("util"))
+  .settings(utilSettings:_*)
+  .withTestSettings
+
 lazy val rawlsModel = project.in(file("model"))
   .settings(modelSettings:_*)
   .withTestSettings
 
+lazy val rawlsGoogle = project.in(file("google"))
+  .settings(googleSettings:_*)
+  .dependsOn(rawlsModel)
+  .dependsOn(rawlsUtil)
+  .withTestSettings
+
 lazy val rawlsCore = project.in(file("core"))
   .settings(rawlsCoreSettings:_*)
   .dependsOn(rawlsModel)
+  .dependsOn(rawlsGoogle)
+  .dependsOn(rawlsUtil)
   .withTestSettings
 
 lazy val rawls = project.in(file("."))
   .settings(rootSettings:_*)
+  .aggregate(rawlsGoogle)
```

Comment:
Definitely run and test assembly here. The difference between aggregate and dependsOn tripped me up a lot.